main.tf

```
@@ -41,6 +41,12 @@ resource "aws_cloudfront_vpc_origin" "this" {
     }
   }
 
+  timeouts {
+    create = try(each.value.timeouts.create, null)
```

these all should not need the `try()` blocks given the variable definition:
```suggestion
    create = each.value.timeouts.create
```

---

fixed in [cbcac4e](https://github.com/terraform-aws-modules/terraform-aws-cloudfront/pull/165/commits/cbcac4e1ff3e1aef705d086b70621059de8a7aea)